src/storageClients/BlobStorageManager.js

```diff
@@ -0,0 +1,89 @@
+'use strict';
+
+const Promise = require('bluebird');
+const azure = Promise.promisifyAll(require('azure-storage'));
+
+const BLOB_STORAGE_CONNECTION_STRING = process.env.BLOB_STORAGE_CONNECTION_STRING;
+
+module.exports = {
+
+  Get: function(containerName, blobName, id) {
+    let blobSvc = azure.createBlobService(BLOB_STORAGE_CONNECTION_STRING);
+    return new Promise((resolve, reject) => {
+      blobSvc.getBlobToTextAsync(containerName, blobName, null)
+        .then(text => {
+          let item = JSON.parse(text)[id];
+          if(item) {
+            resolve(JSON.parse(text)[id]);
+          } else {
+            resolve(null);
```

Comment:
yea that makes sense, I originally thought that it shouldn't return an error since you just found nothing, but if you found nothing then you can't really go on in the process.